.env.example

```
@@ -15,3 +15,13 @@ VERTEX_PROJECT_ID=your-gcp-project-id
 # Optional: Path to service account credentials JSON file (alternative to API key)
 GOOGLE_APPLICATION_CREDENTIALS=/path/to/service-account-credentials.json
+
+# MCP Tool Loading Configuration (Optional)
+# Controls which tools are loaded by the MCP server to optimize token usage
+# Options: "all" (default), "standard", "core", or comma-separated list of tool names
+# Examples:
+#   TASK_MASTER_TOOLS=standard          # ~50% reduction
+#   TASK_MASTER_TOOLS=core              # ~70% reduction
+#   TASK_MASTER_TOOLS=get_tasks,next_task,set_task_status  # Custom selection
+# If not set or empty, defaults to "all"
+# TASK_MASTER_TOOLS=all
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)